app/cli/cmd/root_test.go

```diff
@@ -44,8 +45,9 @@ func TestParseToken(t *testing.T) {
 		},
 		{
 			name:  "api token",
```

Comment:
you should probably try a token that's missing some properties. That's a case that could happen in the future and will happen today (old API tokens without org)